### PR TITLE
Updating README.md file

### PR DESCRIPTION
**Pull request by shivkanya9146**
/cc @mayurkale22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,38 @@ OpenTelemetry Google Cloud Trace Exporter allows the user to send collected trac
 
 [Google Cloud Trace](https://cloud.google.com/trace) is a distributed tracing system. It helps gather timing data needed to troubleshoot latency problems in microservice architectures. It manages both the collection and lookup of this data.
 
+### Getting Started
+
+This exporter package assumes your application is [already instrumented](https://github.com/open-telemetry/opentelemetry-js/blob/master/getting-started/README.md) with the OpenTelemetry SDK. Once you are ready to export OpenTelemetry data, you can add this exporter to your application:
+
+sh
```
Comment (mayurkale22) [inline README.md:13]: please include opening and closing backticks(```)

File: README.md [hunk 2]
```diff
@@ -6,6 +6,38 @@ OpenTelemetry Google Cloud Trace Exporter allows the user to send collected trac
...
+npm install --save @google-cloud/opentelemetry-cloud-trace-exporter
+
+
+Add the exporter to your existing OpenTelemetry tracer provider (`NodeTracerProvider` / `BasicTracerProvider`)
+
+js
```
Comment (mayurkale22) [inline README.md:19]: Same here.

File: README.md (hunk 1 above)
Comment (mayurkale22) [reply README.md:13]: any update?

Comment (shivkanya9146) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (shivkanya9146) [reply README.md]: done.

**Pull request merged**